lk2nd/device/dts/msm8952/msm8940-palm-pepito.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/*
+ * There are a few quirks to be aware of
+ *
+ * - Fastboot is not available on this device, lk2nd must be flashed via EDL.
+ *   Some EDL programs (or versions thereof) do not work properly with available
+ *   firehoses (or else need responses ignored).  QDL is a known working implementation
+ *
+ * - The stock bootloader checks the OS version and the OS patch level in the boot.img as
+ *   well as ensuring there is a signature in the boot.img (it does not matter the signature).
+ *   When compiling lk2nd, set MKBOOTIMG_OS_VERSION=8.1.0 MKBOOTIMG_OS_PATCH_LEVEL=2020-09-01
+ *   and SIGN_BOOTIMG=1
+ */
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8940 0>;
+	qcom,board-id = <0xaa000008 0x00>;
+};
+
+&lk2nd {
+	model = "Palm Phone (pepito)";
+	compatible = "palm,pepito";
+
+	lk2nd,dtb-files = "qcom,msm8940-mtp", "qcom,msm8940", "qcom,mtp";
```

Comment:
This line should define a dtb filename for this device, not a list of compatibles (especially downstream ones), which would be used for booting a generic linux image with extlinux.conf. If you have some work to run linux on this device, please put the name of your dtb here 

https://github.com/msm8916-mainline/lk2nd/blob/4801a1e9e0c0f26c99dce2df9f5dc970fa2cf8e5/Documentation/dt-bindings.md?plain=1#L67-L69